cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -77,6 +77,12 @@
 
   public static final String USER_OVERRIDES = "userOverrides";
 
+  public static final String RUNTIME_ARGS = "runtimeArgs";
```

Comment:
This is not used for getting things from system arguments. Shouldn't define in here